ckanext/blob_storage/plugin.py

```diff
@@ -57,7 +59,7 @@ def update_package_schema(self):
                 toolkit.get_validator('upload_has_sha256'),
                 toolkit.get_validator('upload_has_size'),
                 toolkit.get_validator('upload_has_lfs_prefix')
-                ],
+            ],
```

Comment:
better do not update just the whitespaces in the code base. It will be very difficult to resolve merge conflicts of such changes when we'll update the project with origin.